instrumentation/runtime-metrics/library/src/test/java/io/opentelemetry/instrumentation/runtimemetrics/ScopeUtil.java

```
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.runtimemetrics;
+
+import io.opentelemetry.instrumentation.api.internal.EmbeddedInstrumentationProperties;
+import io.opentelemetry.sdk.common.InstrumentationScopeInfo;
+import java.util.Optional;
+
+class ScopeUtil {
+
+  static final InstrumentationScopeInfo EXPECTED_SCOPE =
+      InstrumentationScopeInfo.builder("io.opentelemetry.runtime-metrics")
+          .setVersion(
+              Optional.ofNullable(
+                      EmbeddedInstrumentationProperties.findVersion(
+                          "io.opentelemetry.runtime-metrics"))
+                  .orElse("unknown"))
```

can we throw exception in test if version is not found?